test/units/TEST-72-SYSUPDATE.sh

```diff
@@ -227,15 +239,26 @@ EOF
     test ! -f "$WORKDIR/xbootldr/EFI/Linux/uki_v1.efi.extra.d/extra.addon.efi"
     test ! -d "$WORKDIR/xbootldr/EFI/Linux/uki_v1.efi.extra.d"
 
-    # Create fourth version, and update through a file:// URL. This should be
+    # Create fourth version, update using updatectl and verify it replaced the
+    # second version
+    new_version "$sector_size" v4
+    systemctl start systemd-sysupdated
+    updatectl check
```

Comment:
If the return value of `updatectl` does not correspond to whether or not there's an update available, you should use the `$SYSUPDATE` binary here. This check is here to make sure that there's an update available. If there's no update available, the test should fail - however `updatectl check` doesn't implement this behavior we want.

Alternatively, we could just make `updatectl check` return failure if no update is available

Reply:
I'd rather it not return failure, as a user I don't think I'd want "no update" to be considered an error.